spec/controllers/questions/consent_controller_spec.rb

```diff
@@ -202,6 +202,31 @@
         expect(error_messages[:primary_last_name].first).to eq "Please enter your last name."
       end
     end
+
+    context "with an itin applicant that has a duplicate" do
+      before do
+        allow(intake).to receive(:has_duplicate?).and_return true
+        allow(intake).to receive(:itin_applicant?).and_return true
+      end
```

Comment:
Trivial & stylistic, non blocking: Our codebase tends to have a blank line between a `before do...end` and a `let...end` and the start of an `it do...end`